### ISSUE TITLE
Uint shift count

### ISSUE THREAD

**Issue by okrcma** [quapka/PA193_mnemonic_aller#4]
Since Go 1.13, the shift count (of shift operators) can by an int. However as this version is fairly new, we suggest to still use the artificial conversion to uint for compatibility with older versions of Go.

**Comment by quapka** [comment]
indeed, forcing the user to upgrade is not the nicest thing, especially if this is the only issue when using previous version. Could you create a PR for this?
